packages/web/test/mocks/tx.ts

```diff
@@ -99,4 +109,14 @@ export const mockSuggestedParam: SuggestedParams = {
 	lastRound: 100,
 	genesisID: "testnet-v1.0",
 	genesisHash: "SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=",
-};
\ No newline at end of file
+};
+
+export const txnReceiptMock = {
+	txID: "1",
+	"confirmed-round": 1,
+	"asset-index": 1,
+	"application-index": 1,
+	txn: {
```

Comment:
This doesn't lookd good. Usally, you shouldn't repeat the root nodes.

Reply:
I think we don't require them now, will remove it